Sources/BraintreeVenmo/BTVenmoError.swift

```diff
@@ -102,6 +107,8 @@ enum BTVenmoError: Error, CustomNSError, LocalizedError {
             return description
         case .fetchConfigurationFailed:
             return "Failed to fetch Braintree configuration."
+        case .ecdDisabled:
+            return "Cannot collect customer data when ECD is disabled."
```

Comment:
```suggestion
            return "Cannot collect customer data when Enriched Customer Data is disabled."
```
Can we add something like "Enable this feature in the Control Panel to collect this data." to the end of this message? That assumes it's self service for merchants to enable. If it's not we should add a not about contact support/sales/whomever is responsible for enabling this feature.

Reply:
Good idea 👍🏼